python/tests/plotting/plot_helper.py

```diff
@@ -4,6 +4,19 @@
 # @author Neil Vaytet
 import numpy as np
 import scipp as sc
+import matplotlib.pyplot as plt
+
+
+def close(fig):
+    if hasattr(fig, 'fig'):
+        plt.close(fig.fig)
+    elif isinstance(fig, dict):
+        for x in fig.values():
+            close(x)
```

Comment:
I wonder if it would be better to make this a method of the `Plot` and `SciPlot` objects?
Something people may find useful to have in notebooks?
Currently we have the `.make_static()` method, but that just sets a flag. But we could tell it to close the figure at the same time?
However, it would be nice if, when we do
```Py
p = sc.plot(...)
p.close()
```
we could still subsequently display a static version of `p` in the notebook when just entering `p` in a cell.
I think closing the figure would cause this to fail.
We could maybe keep a copy of the image in the figure class?

Reply:
What is the purpose of `make_static`? Does it produce an image to embed in the notebook or does it just disable the custom interaction?

Regardless, I agree that a close method would be useful.

Reply:
see here https://github.com/scipp/scipp/pull/1800

Reply:
`make_static` was setting a flag, so that the next time the figure was displayed it would be as a static image. In other words, not a very good implementation, with an obscure name. `close()` is much better i think.